weeklyChallenges01/ChallengesWithTest01.cs

```diff
@@ -8,37 +8,64 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if ( num1 == num2)
```

Comment:
Great use of if/else statements. However, consider that the return type is type `bool` and therefore only two possible choices: true or false.  With knowing that, putting `return num1 == num2;` will either return true if true, vice versa.